ipyleaflet/leaflet.py

```diff
@@ -778,6 +778,16 @@ def clear_markers(self):
         self.send({'msg': 'clear_markers'})
 
```

Comment:
You would need to change the `Map` constructor in this file. Maybe smth like:

```python
def __init__(self, *args, **kwargs):
    zoom_control = kwargs.get('zoom_control', True)
     if zoom_control:
        self.add_layer(ZoomControl())
    [...]
```

So that we keep the old behavior (creating the zoom control by default if `zoom_control is True`)

Reply:
```
 def __init__(self, **kwargs):
        super(Map, self).__init__(**kwargs)
        self.on_displayed(self._fire_children_displayed)
        self.on_msg(self._handle_leaflet_event)
        
        if self.zoom_control:
            self.add_control(ZoomControl())
```
kwargs is used in `super(Map, self).__init__(**kwargs)` so i can call self.zoom_control.